index.js

```diff
@@ -70,21 +71,31 @@ function lint(textEditor) {
 			};
 		}
 
+		// taken from linter-eslint
+		let range;
+		const msgLine = x.line - 1;
+		try {
+			if (typeof x.endColumn === 'number' && typeof x.endLine === 'number') {
+				const msgCol = Math.max(0, x.column - 1);
+				range = [[msgLine, msgCol], [x.endLine - 1, x.endColumn - 1]];
+			} else if (typeof x.line === 'number' && typeof x.column === 'number') {
+				// We want msgCol to remain undefined if it was initially so
+				// `rangeFromLineNumber` will give us a range over the entire line
+				const msgCol = typeof x.column === 'undefined' ? x.column : x.column - 1;
+				range = rangeFromLineNumber(textEditor, msgLine, msgCol);
+			}
+		} catch (err) {
+			throw new Error(`Cannot mark location in editor for (${x.ruleId}) - (${x.message}) at line (${x.line}) column (${x.column})`);
```

Comment:
@jonathandelgado What @sindresorhus means is that `rangeFromLineNumber` throws when it is fed an invalid point, the only way it should be throwing is if an ESLint rule has a bug in it. A great example of this is how `no-multiple-empty-lines` [was broken](https://github.com/eslint/eslint/issues/7086) and returning the wrong point between versions v1.8.0 and v3.4.0. It was fixed in v3.5.0 (due to getting reported in `linter-eslint`).

The problem with a broken rule like that throwing is that it happens on _every edit_ for on the fly linters, hence why we plan to make it less intrusive. I like your idea of pointing users to where to report it as well @sindresorhus!